package.json

```diff
@@ -19,8 +19,8 @@
   },
   "scripts": {
     "watch": "tsc -w",
-    "build": "rm -rf dist; tsc -d",
-    "prepublish": "npm run build; npm test",
+    "build": "rimraf dist; tsc -d",
+    "prepublishOnly": "npm run build; npm test",
```

Comment:
While we're at it.. does it make sense to switch the order of these two? E.g. Why bother building if the tests don't pass.